razorpay-payments.php

```diff
@@ -44,6 +51,10 @@ public function __construct()
             $this->key_secret = $this->settings['key_secret'];
             $this->payment_action = $this->settings['payment_action'];
 
+            $this->enable_webhook = $this->settings['enable_webhook'];
```

Comment:
This broke the last time we did a release with `payment_action`. We had to ask users to visit settings and submit it again so it entered the database. Have a default value here, so it doesn't give errors.